share/availability/full/reconstruction_test.go

```diff
@@ -260,4 +265,5 @@ func TestShareAvailable_DisconnectedFullNodes(t *testing.T) {
 	require.NoError(t, err, share.ErrNotAvailable)
 	err = full2.SharesAvailable(ctx, root)
 	require.NoError(t, err, share.ErrNotAvailable)
+	wg.Wait()
```

Comment:
Why? We don't really care if the shares available end for lights. Or was there a panic about test fail in the routine after the test ended?

Reply:
Also, it would be nice to extract this into separate PR

Reply:
This test panicing now. The panic is caused by error log happening after test has exit. I just don't like panicing tests in main.

Reply:
Is it constantly panicing? We can also merge the extracted PR first before this one

Reply:
it panics in CI only, but did it 3 out of 3 times